docs/source/model_design/vivarium_model_components/risk_factors/risk_correlation/bias_quantification.ipynb

```diff
@@ -0,0 +1,1387 @@
+{
```

Comment:
<p>Typo: *associated</p><p> </p><p>I think it would be clearer to remove the outcome bubble from this diagram, as it is not included in the below functions.</p><p> </p><p>Let's add a note here: in this situation, we normally do not have to worry about PAFs, because our baseline intervention coverage is 0. When baseline intervention coverage is not zero, a "joint PAF" of R1 and R2 on the intervention would need to account for correlation. This <strong>cannot</strong> be done using the method described on the risk-risk correlation page, because there is an interaction effect/effect modification between R1 and R2. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ihmeuw/vivarium_research/pull/1113/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>